userguide/virtualmachines.rst

```diff
@@ -44,85 +44,112 @@ output shows the POPCNT feature, this processor can be used with
    processors ceased in 2012 or 2013.
 
 
-.. index:: Creating VMs
-.. _Creating VMs:
+.. index:: Virtual Machines Page
+.. _Virtual Machines Page:
 
-Creating VMs
-------------
+Virtual Machines Page
+---------------------
 
-Click on
+Go to
 :menuselection:`Virtual Machines`
-and |ui-add| to open the wizard shown in
-:numref:`Figure %s <vms_add_fig>`:
+to see a table with each installed VM. A graph of total VM memory usage
+displays at the top of the page.
+
+.. figure:: images/virtual-machines.png
 
+   Virtual Machines Page
 
-.. _vms_add_fig:
 
-.. figure:: images/virtual-machines-add-wizard-type.png
+The VM :guilabel:`Name`, :guilabel:`State`, :guilabel:`VNC Port` (if
+present), :guilabel:`Type`, :guilabel:`Description`, number of
+:guilabel:`Virtual CPUs`, :guilabel:`Memory Size` (in MiB),
+:guilabel:`Boot Loader Type`, and :guilabel:`Autostart` values are shown
+in the table. The table display is adjusted by clicking
+:guilabel:`COLUMNS` and selecting which columns are shown in the table.
 
-   Add VM
+Click |ui-options| to see additional buttons for a VM.
 
+These choices are available in the |ui-options| menu regardless of the
+current :guilabel:`State`:
 
-Create a virtual machine by following these steps in the creation
-wizard:
+* :guilabel:`Edit` opens a form to change the VM settings. The VM name,
```

Comment:
```suggestion
* :guilabel:`Edit` changes the VM settings.
```
Again, we don't need to list all the fields that can be edited.  That will be visible on the screen.  Also, it will change from version to version, so listing them here is just more work to keep in sync with no real benefit to the user.